learning_assistant/utils.py

```diff
@@ -47,30 +49,37 @@ def get_reduced_message_list(prompt_template, message_list):
         # insert message at beginning of list, because we are traversing the message list from most recent to oldest
         new_message_list.insert(0, new_message)
 
-    system_message = {'role': 'system', 'content': prompt_template}
-
-    return [system_message] + new_message_list
+    return new_message_list
 
 
 def create_request_body(prompt_template, message_list):
     """
     Form request body to be passed to the chat endpoint.
     """
+    messages = get_reduced_message_list(prompt_template, message_list)
+
     response_body = {
-        'message_list': get_reduced_message_list(prompt_template, message_list),
+        'message_list': [{'role': 'system', 'content': prompt_template}] + messages,
     }
 
+    if v2_endpoint_enabled():
+        response_body = {
+            'client_id': getattr(settings, 'CHAT_COMPLETION_CLIENT_ID', 'edx_olc_la'),
```

Comment:
Question: what is the 'edx_olc_la'? Is there a reason why this is hardcoded rather than added as a setting?